Package.swift

```diff
@@ -0,0 +1,42 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "JustRideSDK",
+    platforms: [
+        .iOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "JustRideSDK",
+            targets: ["JustRideSDKTarget"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ale-gen/RNCryptor.git", exact: "5.1.0"),
+        .package(url: "https://github.com/ale-gen/MarqueeLabel.git", exact: "4.3.2"),
+        .package(url: "https://github.com/ale-gen/ZipArchive.git", exact: "2.4.3"),
+        .package(url: "https://github.com/ale-gen/zxingify-objc.git", exact: "3.6.7")
+    ],
+    targets: [
+        .target(
+            name: "JustRideSDKTarget",
+            dependencies: [
+                .target(
+                    name: "JustRideSDK",
+                    condition: .when(platforms: [.iOS])
```

Comment:
If you remove this line, does anything break?

Reply:
This line just explicitly tells for which platforms dependency should be applied

Reply:
Okay, let's discuss after standup. I'll merge this PR in as it is for now.